cmd/encryption-v1.go

```diff
@@ -896,6 +913,57 @@ func (o *ObjectInfo) DecryptedSize() (int64, error) {
 	return size, nil
 }
 
+// For encrypted objects, the ETag sent by client if available
+// is stored in encrypted form in the backend.Decrypt the ETag
```

Comment:
```suggestion
// is stored in encrypted form in the backend. Decrypt the ETag
```

It just looks like a code snippet without the space.